client/src/components/DateTime/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import styles from "./style.module.scss";
+
+function DateTime() {
+    const [dateObject, setDateObject] = useState(new Date());
+
+    useEffect(() => {
+        let interval = setTimeout(() => setDateObject(new Date(), 1000 * 60));
```

Comment:
There should be a `return () => clearTimeout(interval)` here

Reply:
Just for curiousity, what does the return statement needs to be there?

Reply:
The returned function will be called when the component unmounts, so it will clear the timeout so that the `dateObject` state isn't attempted to be set after the component has already unmounted

Reply:
Sweet, thanks @sannan06 !